ibis/expr/operations.py

```diff
@@ -720,6 +720,18 @@ class Quantile(Reduction):
     output_type = rules.type_of_arg(1)
 
 
+class MultiQuantile(Reduction):
+
+    input_type = [value,
+                  rules.collection(name='quantile'),
```

Comment:
You want `rules.array(dt.double)` here (`dt.double` for now, we'll extend as needed).